fendermint/vm/interpreter/src/fvm/state/exec.rs

```diff
@@ -177,11 +176,13 @@ where
 
     /// Execute message implicitly.
     pub fn execute_implicit(&mut self, msg: Message) -> ExecResult {
+        // TODO Karel - measure here
```

Comment:
Do we _really_? I think it's fine not to overload this component with instrumentation, and just leave it up to the user -- just like we're doing now.

Reply:
Not really yeah :) I forgot to remove my comment. It should not be there..